packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -86,7 +86,7 @@ library TaikoData {
     /// @dev Struct representing input data for block metadata.
     struct BlockMetadataInput {
         bytes32 txListHash;
-        address beneficiary;
+        address proposer;
```

Comment:
`beneficiary` seems like the better name because it's the fee recipient address, not the proposer?